apps/survey/contracts/Survey.sol

```diff
@@ -244,6 +256,7 @@ contract Survey is AragonApp {
         return surveys[_surveyId].metadata;
     }
 
+    /* solium-disable function-order */
```

Comment:
We should probably put this at the top, or disable it for a particular line?

This declaration will disable the rule from this point on in the file.